LoginServices/LoginMainActivity.py

```
@@ -0,0 +1,46 @@
+from CookiesSetup.CookiesSetup import CookiesSetup
+from LoginServices.loginServices import LoginServices
+from flask import jsonify
+
+class LoginMainActivity:
+      def __init__(self):
+          self.cookies=CookiesSetup()
+          self.loginServices=LoginServices()
+      def authenticateUser(self,user_data):
```

Flake8 (7.2.0)

[error] expected 1 blank line, found 0

(E301)